src/components/ComparisionChart/functionMapper.ts

```diff
@@ -44,6 +44,14 @@ const mapper = {
 // ---Functions for data mapping of Demographic data---
 // Stack Bar Chart Mapper starts
 export const DemographicsStackAgeGetter = (chartObj: chartObj, data: any) => demographicsStackGetter(data, chartObj, 'age');
+export const DemographicsStackGenderGetter = (chartObj: chartObj, data: any) => demographicsStackGetter(data, chartObj, 'gender');
+export const DemographicsStackIncomeGetter = (chartObj: chartObj, data: any) => demographicsStackGetter(data, chartObj, 'income');
```

Comment:
please remove this file as wellas the below one.
If you want to add it for a test do it in unit tests